lib/ui/form/form_screen.dart

```diff
@@ -109,17 +130,88 @@ class FormScreenState extends ConsumerState<FormScreen> {
     );
   }
 
-  Widget _buildStartButton() {
-    return Padding(
-      padding: const EdgeInsets.all(AppDimensions.spacing20),
+  Widget _buildCloseButton(BuildContext context) {
+    return Visibility(
+      visible: _showCloseButton,
       child: Align(
-        alignment: Alignment.bottomRight,
-        child: ElevatedButton(
+        alignment: Alignment.topRight,
+        child: RawMaterialButton(
+          shape: const CircleBorder(),
+          fillColor: AppColors.white20,
+          constraints: const BoxConstraints.tightFor(
+            width: AppDimensions.closeButtonSize,
+            height: AppDimensions.closeButtonSize,
+          ),
+          child: const Icon(
+            Icons.close,
+            color: Colors.white,
+            size: AppDimensions.closeButtonIconSize,
+          ),
           onPressed: () {
-            // TODO: Integrate click-event from survey details #25
+            _appNavigator.navigateBack(context);
           },
-          child: Text(
-            AppLocalizations.of(context)!.start_survey,
+        ),
+      ),
+    );
+  }
+
+  Widget _buildStartSurveyButton() {
```

Comment:
As `_buildStartSurveyButton()`, `_buildNextSurveyButton()`, `_buildNextSurveyButton()` only have one return statement we can use short hand `=>` to replace `return`.